app/views/campaigns/virtual_host.txt.erb

```diff
@@ -6,5 +6,9 @@
     DirectoryIndex <%= @directory_index %>
     CustomLog     <%= @approot %>/log/www-campaign-<%= @campaign_id %>-access.log combined
     ErrorLog      <%= @approot %>/log/www-campaign-<%= @campaign_id %>-error.log
+
+    ProxyPass /reports/image <%= GlobalSettings.first.site_url %>/reports/image
```

Comment:
This may need the following directives if the base site is running over HTTPS:

```
   SSLProxyEngine Off
    SSLProxyCheckPeerCN Off
    SSLProxyCheckPeerExpire Off
```

Could do a check to see if the URL contains `https://`